js/index.js

```diff
@@ -30,25 +33,72 @@ app.controller("IndexController", ["$scope", "$http", function($scope, $http) {
     var urlString = "http://nominatim.openstreetmap.org/reverse?format=json&lat=" +
     latlng.lat + "&lon=" +
     latlng.lng + "&zoom=18&addressdetails=1";
-    $http.get(urlString).then(addMarker);
+    $http.get(urlString).then(addMarker,errorMessage);
   }
 
   function addMarker(response) {
+
     var marker = {
       lat: parseFloat(response.data.lat),
       lng: parseFloat(response.data.lon),
-      message: "New entry",
+      title: "New entry",
       dueDate: new Date(),
       address: response.data.display_name
     };
 
-    $scope.markers.push(marker);
+
+
     $scope.currentMarker = marker;
     $("#modal").modal('show');
+    $("#modal").on('click','#btnClose',function(){
+      //console.log(marker[]);
+      $http.post(serverUrl,marker).then(postSuccess,errorMessage);
+      marker = {};
+    })
   }
 
+
+  $scope.remove = function(index) {
+    if (index != -1) {
```

Comment:
👍 Does this fix the issue when deleting the last item?

Reply:
I modified and solved it, but I guarantee it is not the best way. :-P